command/server/config.go

```diff
@@ -102,6 +103,7 @@ func DefaultConfig() *Config {
 		EnableGraphQL:            false,
 		JSONRPCBatchRequestLimit: jsonrpc.DefaultJSONRPCBatchRequestLimit,
 		JSONRPCBlockRangeLimit:   jsonrpc.DefaultJSONRPCBlockRangeLimit,
+		EnableWS:                 false,
```

Comment:
default set true.

Reply:
Set `true` as default would violate the limit purpose.